react-ui/src/pages/contest/ContestUnderReview.jsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * Displayed between submission and voting, which can be delayed
+ */
+
+import Box from '@material-ui/core/Box';
+import PropTypes from 'prop-types';
+
+import {
+  SpinnerButton,
+} from '../../components';
+
+function ContestUnderReview({ isValidating, mutate }) {
+  return (
+    <>
+      <Box marginBottom={2}>
+        We are working on getting the contest ready! Please check again soon.
+      </Box>
+      <SpinnerButton
+        color="primary"
+        disabled={isValidating}
+        onClick={mutate}
+        submitting={isValidating}
+        variant="contained"
+      >
+        Refresh
+      </SpinnerButton>
+    </>
+  );
+}
+
+ContestUnderReview.propTypes = {
+  isValidating: PropTypes.bool,
+  mutate: PropTypes.func.isRequired,
+};
+
+ContestUnderReview.defaultProps = {
+  isValidating: true,
```

Comment:
I switched to `false` but kept it as a prop. Eventually I think we will have multiple hooks to grab slices like `isValidating` from stored state, otherwise I think it's 6 lines of code to get `useSwrData`.